client/lib/importer/importer-config.js

```diff
@@ -137,6 +137,60 @@ function getConfig( { siteTitle = '' } = {} ) {
 		weight: 0,
 	};
 
+	importerConfig.substack = {
+		engine: 'substack',
+		key: 'importer-type-substack',
+		type: 'file',
+		title: 'Substack',
+		icon: 'substack',
+		description: translate(
+			'Import posts and images, podcasts and public comments from a %(importerName)s export file to {{b}}%(siteTitle)s{{/b}}.',
```

Comment:
I was reusing the existing pattern in the file, but you're right - since the description is different and likely will be unique to Substack we can remove the placeholder

Reply:
I'm inclined to keep this as is, as:
1. It seems like a nit, and the there is code to substitute importer name anyways. If the service name changes for e.g. this might make things simpler.
2. It makes this code not similar to the previous ones, and if I make a change, I'd rather it be everywhere, in a separate PR.